Directory.Packages.props

```
@@ -4,12 +4,14 @@
     <CentralPackageVersionOverrideEnabled>false</CentralPackageVersionOverrideEnabled>
   </PropertyGroup>
   <ItemGroup>
-    <PackageVersion Include="Appveyor.TestLogger" Version="2.0.0" />
     <PackageVersion Include="BenchmarkDotNet" Version="0.14.0" />
     <PackageVersion Include="BouncyCastle.Cryptography" Version="2.4.0" />
     <PackageVersion Include="coverlet.collector" Version="6.0.2" />
     <PackageVersion Include="coverlet.msbuild" Version="6.0.2" />
-    <PackageVersion Include="LiquidTestReports.Markdown" Version="1.0.9" />
+    <PackageVersion Include="GitHubActionsTestLogger" Version="2.4.1">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageVersion>
```

are these bits needed here?
```suggestion
    <PackageVersion Include="GitHubActionsTestLogger" Version="2.4.1"/>
```

---

They may not strictly be needed, but there are set by default for this package. Imho `PrivateAssets` makes sense because the test code itself doesn't actually need this dependency, only the dotnet test host does.